tools/bin/mbedtls-release-checker.py

```diff
@@ -0,0 +1,474 @@
+                                        self.logger.log(logging.INFO,
+                                                        "Forcing {} to be backport on link".format(target_link_pr.pr_number))
+                                break
+
+    def get_cvs_link(self, link_target, link_text = ''):
+
+        if link_text == '':
+            return '\"=HYPERLINK(\"{}\")\"'.format(link_target)
+        else:
+            return '=HYPERLINK(\"{}\",\"{}\")'.format(link_target, link_text)
+
+
+    def write_csv(self, file_name):
+
+        self.logger.log(logging.INFO, 'Writing CSV file to {}'.format(file_name))
+
+        header_fields = ['Hash', 'PR Number', 'Title', 'Is Backport', 'Has Changelog',
+                         'Changelog Check', 'Backport Check', 'Tests Check', 'Backport Link',
+                         'Linked PRs+']
+
+        with open(file_name, 'w') as csv_file:
+
+            csv_writer = csv.writer(csv_file)
+
+            csv_writer.writerow(header_fields)
+
+            for hash, pr in self.github_prs.items():
+
+                backport_num = pr.backport_of if pr.is_backport else pr.backport
+
+                if backport_num == '':
+                    backport = 'None'
+                else:
+                    backport = self.get_cvs_link('https://github.com/{}/pull/{}'.format(self.github_repo,
+                                                                                        backport_num),
+                                                 backport_num)
+
+                csv_row = [pr.hash, self.get_cvs_link(pr.get_github_link(), pr.pr_number),
+                           pr.title, pr.is_backport, pr.has_changelog, pr.changelog_check,
+                           pr.backport_check, pr.tests_check, backport]
+
+                linked_prs = ''
+                first = True
+
+                for link_pr in pr.linked_prs:
+                    csv_row.append(self.get_cvs_link('https://github.com/{}/pull/{}'.format(self.github_repo,
+                                                                                            link_pr),
+                                                     link_pr))
+
+                csv_writer.writerow(csv_row)
+
+def main():
+
+    parser = argparse.ArgumentParser(description='Attempt to automatically detect missing backports/Changelog entries for an MbedTLS release')
+    parser.add_argument('LTS_range_start', help='Git hash/label of LTS branch range start')
+    parser.add_argument('LTS_range_end', help='Git hash/label of LTS branch range start')
+    parser.add_argument('dev_range_start', help='Git hash/label of development branch range start')
+    parser.add_argument('dev_range_end', help='Hash/label of development branch range end')
+    parser.add_argument('-lo', '--LTS_output', help='Ouput file to dump LTS release csv data to')
+    parser.add_argument('-do', '--dev_output', help='Ouput file to dump development release csv data to')
```

Comment:
I would have predefined defaults for those files. This script takes a significant ammount and API calls to go through a run, that will do nothing in the current default configuration. 

It would make more sense for the user to explicitly turn off output file generation.